strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/layout/nuget/NugetArtifactControllerTest.java

```diff
@@ -280,14 +276,10 @@ public void testChocolatey(@NugetRepository(storageId = STORAGE_ID,
         final String storageId = repository.getStorage().getId();
         final String repositoryId = repository.getId();
         final String packageId = "Org.Carlspring.Strongbox.Examples.Nuget.Mono-Test";
-        final String packageVersion = "1.0.0";
-
-        long packageSize = Files.size(packagePath);
-        byte[] packageContent = readPackageContent(packagePath);
 
         // Push
         String url = getContextBaseUrl() + "/storages/{storageId}/{repositoryId}/";
-        createPushRequest(packageContent).when()
+        createPushRequest(packagePath).when()
                                          .put(url, storageId, repositoryId)
```

Comment:
Cosmetic: Could you please re-indent the chained methods? Thanks! :)

Reply:
Fixed.